crates/re_log/Cargo.toml

```diff
@@ -31,4 +31,4 @@ env_logger = "0.10"
 # web dependencies:
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 js-sys = "0.3"
-wasm-bindgen = "=0.2.84"
```

Comment:
Using `"=…` here and below was a mistake. We have a `Cargo.lock` file for that. We don't want force any potential users of our crates to use a specific wasm-bindgen version. However, we do want to force them to use _at least_ 0.2.86 since it contains the critical fix for 2GiB RAM usage